lib/matplotlib/backends/backend_macos_common.py

```diff
@@ -0,0 +1,10 @@
+import sys
```

Comment:
this should definitely be a private module (with a leading underscore)

Reply:
rename the file to e.g. _macos_helpers.py or whatnot.

Reply:
Gotcha, will do.

Reply:
See latest push.